clickhouse-client/src/main/java/com/clickhouse/client/ClickHouseNodes.java

```diff
@@ -95,14 +96,19 @@ static ClickHouseNodes create(String endpoints, Map<?, ?> defaultOptions) {
             }
 
             int endIndex = i;
+            // parsing host name
```

Comment:
I do not fully understand if we are planning to remove this module. 
why we are changing `ClickHouseNodes`